spec/models/grant_spec.rb

```diff
@@ -287,4 +287,39 @@
     end
 
   end
+
+  describe '#methods' do
+    let(:grant)     { create(:published_open_grant_with_users, :with_reviewer) }
+    let(:new_user)  { create(:user) }
+
+    context 'roles' do
```

Comment:
This is a good place for shared examples.

Reply:
since these are testing different, dynamically-created methods, i'm going to leave them as is